lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -1,63 +1,72 @@
+                <% } %>
+            </div>
+            <div class="setup-actions">
+                <% if (studioAdvancedSettingsUrl !== "None") { %>
+                    <a href="<%= studioAdvancedSettingsUrl %>" class="action-secondary action-edit"><%- gettext("Edit settings in Studio") %></a>
+                <% } %>
+            </div>
         </div>
-    </div>
-</header>
+    </header>
 
-<!-- individual group - form -->
-<div class="cohort-management-group-add">
-    <form action="" method="post" id="cohort-management-group-add-form" class="cohort-management-group-add-form">
+    <ul class="wrapper-tabs">
+        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><%- gettext("Manage Students") %></a></li>
+        <li class="tab tab-settings" data-tab="settings"><a href="#"><%- gettext("Settings") %></a></li>
+    </ul>
 
-        <h4 class="form-title"><%- gettext('Add students to this cohort group') %></h4>
+    <div class="cohort-management-group-add tab-content tab-content-manage_students">
+        <form action="" method="post" id="cohort-management-group-add-form" class="cohort-management-group-add-form">
 
-        <div class="form-introduction">
-            <p><%- gettext('Note: Students can only be in one cohort group. Adding students to this group overrides any previous group assignment.') %></p>
-        </div>
+            <h4 class="form-title"><%- gettext('Add students to this cohort group') %></h4>
+
+            <div class="form-introduction">
+                <p><%- gettext('Note: Students can be in only one cohort group. Adding students to this group overrides any previous group assignment.') %></p>
+            </div>
 
-        <div class="cohort-confirmations"></div>
-        <div class="cohort-errors"></div>
+            <div class="cohort-confirmations"></div>
+            <div class="cohort-errors"></div>
 
-        <div class="form-fields">
-            <div class="field field-textarea is-required">
-                <label for="cohort-management-group-add-students" class="label">
-                    <%- gettext('Enter email addresses and/or usernames separated by new lines or commas for students to add. *') %>
-                    <span class="sr"><%- gettext('(Required Field)') %></span>
-                </label>
-                <textarea name="cohort-management-group-add-students" id="cohort-management-group-add-students"
-                          class="input cohort-management-group-add-students"
-                          placeholder="<%- gettext('e.g. johndoe@example.com, JaneDoe, joeydoe@example.com') %>"></textarea>
+            <div class="form-fields">
+                <div class="field field-textarea is-required">
+                    <label for="cohort-management-group-add-students" class="label">
+                        <%- gettext('Enter email addresses and/or usernames separated by new lines or commas for students to add. *') %>
```

Comment:
"Enter email addresses and/or usernames, separated by new lines or commas, for the students you want to add."